### PR TITLE
[ANDRIY_ZHUKEVYCH]Tasks 21-30

### PR DESCRIPTION
**Pull request by Hollister009**

### CODE REVIEW

File: tasks/30_people-in-bus/people-in-bus.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const peopleInBus = (bus) => {
+  let peopleInBus = 0;
+  bus.forEach((stop) => {
```
Comment (AlyonaPianykh) [inline tasks/30_people-in-bus/people-in-bus.js]: Also you can use destructuring from ES6 to make code look better

```
bus.forEach(( [getIn, getOut]) => {
    inBus += getIn;
    inBus -= getOut;
...
```
See the link: `https://dev.to/sarah_chima/destructuring-assignment---arrays-16f`

Comment (tarasbobak) [reply tasks/30_people-in-bus/people-in-bus.js]: why not a single peopleInBus = peopleInBus + stop[0] - stop[1]?

File: tasks/21_check-letters/check-letters.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const checkLetters = (arr) => {
+  let flag = true;
+  const word = arr[0].toLowerCase();
+  const letters = arr[1].toLowerCase().split('');
+  letters.every((letter) => {
```
Comment (tarasbobak) [inline tasks/21_check-letters/check-letters.js]: Wrong use of every.

Comment (tarasbobak) [reply tasks/21_check-letters/check-letters.js]: Every itself returns true or false in the end
In your code you don't need to store flag in variable at all

File: tasks/24_index-to-insert/index-to-insert.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export const getIndexToInsert = (array, element) => {
+  let modifiedArr = array.slice();
```
Comment (tarasbobak) [inline tasks/24_index-to-insert/index-to-insert.js]: This solution is inefficient.
Array is already sorted.
You generally have to iterate over array and find first el which is bigger than the one you want to insert. And then insert before it.

Comment (Hollister009) [reply tasks/24_index-to-insert/index-to-insert.js]: How this is sorted ?
getIndexToInsert([2, 20, 10], 19)
getIndexToInsert([3, 10, 5], 3)

**Comment by tarasbobak** [comment]
Good job!

**Pull request merged**